test/testUtils/mayaUtils.py

```diff
@@ -135,6 +135,9 @@ def isHydraRenderer():
 def openTestScene(*args):
     filePath = testUtils.getTestScene(*args)
     cmds.file(filePath, force=True, open=True)
+    # To err on the safe side, disable color management in case the scene had color management on.
+    # If/when we add color management tests, they will need to re-enable it manually.
+    cmds.colorManagementPrefs(edit=True, cmEnabled=False)
```

Comment:
After starting work on adding tests for the grid and background color, there are a couple things that happen in the `snapshot` method that I think we could move to instead happen after each scene opening/creation, for example here disabling color management, but also things like disabling the grid, setting a default background color, etc. 

This'll keep the current behavior in the usual cases, but make it easier to change these settings should we need to (which we will for testing them). It'd also make the snapshot method be more of an actual snapshot of the current viewport, rather than it modifying a bunch of things unknowingly before actually taking the snapshot. 

Currently some of these settings are passed in as parameters to the `snapshot` function, but it doesn't really feel to me like that's where they should be, and I feel it's a bit awkward. To make sure those "post-scene opening" settings get applied consistently, I was thinking of maybe adding a script job or subscribing to the MSceneMessages. 

That being said this is all for a future PR : for this current PR, I think this will suffice. I would prefer getting the color management setup functional first as it is going to be in contention with other PRs that may arise, and then refactor afterwards.

Reply:
Isn't this doing a bit too much?  What if I really do want to have a particular color management setup in my file?  I'm also slightly uncomfortable with unconditionally setting multi-sampling.  I would expect that openTestScene() would open my file, not mess with it...

Perhaps we need a new function which does this, openTestSceneApplyTestSettings(), or an optional argument to openTestScene(), applyTestSettings=False?

Reply:
Yeah there's no silver bullet here, so here are a couple thoughts :
1. I agree that `openTestScene()` would ideally just open a test scene. But in the same spirit, `snapshot` should also just take a snapshot, and not mess with the settings, which is what it currently does. However, the risk here is that if we make both of these methods not change anything in the scene, then we depend on test scenes being saved with the right settings, which while is the theoretically correct way to do things, seems to me like it would make it easy to miss if some settings weren't actually set properly. If we go this route, we'll need to make sure we pay very close attention to the scene files.
2. Following that, my idea was that we'd make `snapshot` not change the settings, and on each file open, we'd set those default settings, and if a test requires different values, it would then re-set them to the values it wants. For color management specifically, we're not changing the entire setup, we're just toggling it, so a test would just have to re-enable it. Also, even without this specific instance of disabling color management, currently it'll still be disabled by default in the `snapshot` method, which means you still wouldn't be able to test the color management setup, unless we add a color management status parameter to `assertSnapshotClose` (and if we add a parameter for color management, we should add one for the other settings like hud, grid, etc., which feels a bit misplaced and part of why I want to refactor this).
3. Allowing tests to opt-out of those default settings sounds like a good compromise if we ever need a specific scene setup (though I probably wouldn't have it be a parameter in `mayaUtils.openTestScene()`, since I think they should be applied on new files as well).
4. Yes I also think MSAA should be off as a default. Possibly it was set to on because Storm had it enabled by default, and our existing tests all used it?

Reply:
I like (1) best, it's slightly riskier, but I'd rather have people be deliberate about their settings, and paying attention to your data is good, but this is open to a team discussion.  For the moment I wouldn't change anything and leave this pull request as is.

Reply:
I like the Pierre's idea to be more flexible, probably we can have an applySettings() that can be used in OpenScene() and NewScene()?
Is it possible to have a default settings (E.g., MSAA=ON, CM=OFF, etc.) and a skip flag in our base test class or util?
So in the concrete tests, we can override the default settings (E.g., MSAA=OFF, CM=ON) Or enable skip flag to use scene settings directly if you want?